MediaBrowser.Providers/Plugins/Tmdb/Movies/TmdbMovieProvider.cs

```diff
@@ -48,8 +48,9 @@ public async Task<IEnumerable<RemoteSearchResult>> GetSearchResults(MovieInfo se
             if (tmdbId == 0)
             {
                 var movieResults = await _tmdbClientManager
-                    .SearchMovieAsync(searchInfo.Name, searchInfo.MetadataLanguage, cancellationToken)
+                    .SearchMovieAsync(searchInfo.Name, searchInfo.Year ?? 0, searchInfo.MetadataLanguage, cancellationToken)
```

Comment:
Why not pass the `null` through to the `SearchMovieAsync`? Easier to understand.

I'd also like to avoid the off-chance that the `0` gets included in the query to TMDB. Ex. no results will come back:
![image](https://user-images.githubusercontent.com/6550543/108429088-8798a000-720d-11eb-8916-8611a5d2e845.png)

Reply:
> Why not pass the null through to SearchMovieAsync?

Because you can't, it accepts an `int`, not an `int?`

> I'd also like to avoid the off-chance that the 0 gets included in the query to TMDB. Ex. no results will come back:

```
        public Task<IReadOnlyList<SearchMovie>> SearchMovieAsync(string name, string language, CancellationToken cancellationToken)
        {
            return SearchMovieAsync(name, 0, language, cancellationToken);
        }
```
Before it passed `0` by default.

Reply:
Ahhh, right.

---

Also noticed this logic.
https://github.com/LordMike/TMDbLib/blob/9ad6a80f3ac301aaca16bd0dbb0b4ec1779e4b7c/TMDbLib/Client/TMDbClientSearch.cs#L24-L25